lib/tests/group.rs

```diff
@@ -242,8 +291,8 @@ async fn select_multi_aggregate() -> Result<(), Error> {
 		CREATE test:2 SET group = 1, one = 4.7, two = 3.9;
 		CREATE test:3 SET group = 2, one = 3.2, two = 9.7;
 		CREATE test:4 SET group = 2, one = 4.4, two = 3.0;
-		SELECT group, math::sum(one) AS one, math::sum(two) AS two FROM test GROUP BY group;
-		SELECT group, math::sum(two) AS two, math::sum(one) AS one FROM test GROUP BY group;
+		SELECT group, math::sum(one) AS one, math::sum(two) AS two, math::min(one) as min FROM test GROUP BY group;
+		SELECT group, math::sum(two) AS two, math::sum(one) AS one, math::max(two) as max, math::mean(one) as mean FROM test GROUP BY group;
```

Comment:
Probably we'd better have EXPLAINs here too similarly to we did around L22, because we don't cover EXPLAIN for these math functions in this group.rs tests?

Reply:
Agree! Done